WORKSPACE

```diff
@@ -1,6 +1,8 @@
 workspace(name = "com_github_mvukov_rules_ros2")
 
-load("//repositories:repositories.bzl", "ros2_repositories")
+load("//repositories:repositories.bzl", "ros2_repositories", "rules_ros2_workspace_deps")
+
+rules_ros2_workspace_deps()
```

Comment:
Well yes that's possible, but it would have to be changed back for Bzlmod -- #238 continues to use `ros_repositories()` for ROS 2 repos, but all the other repos like googletest, asio, websocketpp, ... should be comoing from bzlmod directly.
I think it makes sense to have them separate.

If we really want to keep full backwards compatibility and not require WORKSPACE users to update which setup method is used, we can keep the `ros2_repositories` name and split out the ROS 2 repos into another macro that will be called directly by the Bzlmod extension. Not a huge fan of this option though because I think the `ros2_repositories` name would be a misleading.